runtime/onert/api/nnfw/src/nnfw_session.cc

```diff
@@ -1000,7 +1001,17 @@ NNFW_STATUS nnfw_session::set_signature_run(const char *signature)
     return NNFW_STATUS_INVALID_STATE;
   }
 
-  std::cerr << "Error during nnfw_session::set_signature_run : NYI" << std::endl;
+  for (auto &sig : _signature_map)
```

Comment:
(Optional) It would be better to use `const` and structure bindings.
```suggestion
  for (const auto &[subg_idx, sig] : _signature_map)
```